Car.rb

```diff
@@ -0,0 +1,64 @@
+class Car
+  attr_accessor :make, :model, :year, :color, :seats,:owner, :running
+  def initialize(make,model,year,color,seats)
+
+    @make = make
+    @model = model
+    @year = year
+    @color = color
+    @seats = seats
+    @owner='manufacturer'
+    @running = false
+
+  if @make == nil || @model == nil || @year == nil || @color == nil || @seats == nil
+     p "car can't be instantiated, required parameters are not given"
```

Comment:
this condition should be before you start assigning each of the properties

Reply:
and inside the function body, u should return instead of just logging it to console